.travis.yml

```diff
@@ -15,15 +12,18 @@ cache:
 
 env: SYMFONY_VERSION=3.2.*
 
+global:
```

Comment:
According to https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml, the global key must be under the env key:
```yaml
env:
    global:
        - SYMFONY_VERSION=3.2.*
        - SYMFONY_PHPUNIT_DIR=.phpunit
        - SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
        - SYMFONY_PHPUNIT_VERSION=5.6
```